docs/src/content/quickstart.md

```diff
@@ -4,4 +4,35 @@ name: Quickstart
 
 # Quickstart Guide
 
-This will lead you to different paths.
+This Quickstart tutorial will guide you to setup the Hydrus server, we will be using docker in this tutorial,
+Therefore you can go through the [Docker container engine](https://docker-curriculum.com/) and its best practices deploying virtualized environments.
+
+## Running hydrus server
+> Make sure you have docker-compose installed in your system
+
+#### Getting the [hydrus repository](https://github.com/HTTP-APIs/hydrus)
+```bash
+git clone https://github.com/HTTP-APIs/hydrus
+```
+#### Change directory and run docker-compose
+```bash
+cd hydrus
+
+docker-compose up --build
+```
+> If you are using linux you may have to use sudo command with docker
```

Comment:
> If you are using linux you may have to use sudo command with docker

Is this statement verified?

Reply:
https://docs.docker.com/engine/install/linux-postinstall/

The Docker daemon always runs as the root user(default)
we can run it rootless with some extra installations.